workspace.yml

```diff
@@ -85,6 +85,7 @@ attributes:
         DB_PASS: = @('database.pass')
         DB_NAME: = @('database.name')
         DB_DRIVER: = @('database.platform')
+        NEW_RELIC_ENABLED: "false"
```

Comment:
There's no reading of this env var in the commit. Is it the newrelic library reading it? We'll need it to be also not enabled if NEW_RELIC_ENABLED is not supplied

Reply:
It is recognized by the new relic agent library, see https://github.com/newrelic/go-agent/blob/master/v3/newrelic/config_options.go#L392

And it is enabled by default, see https://github.com/newrelic/go-agent/blob/master/v3/newrelic/config.go#L577